common/types/testdata/domain.go

```diff
@@ -88,6 +88,14 @@ var (
 				State: types.IsolationGroupStateDrained,
 			},
 		},
+		AsyncWorkflowConfig: &types.AsyncWorkflowConfiguration{
```

Comment:
hmm.  so there wasn't previously data, and adding it + mapping it -> no tests need to change?
that seems to imply the tests are seeing it _not_ propagate correctly, as they had to have empty values before, yea?

or do we _only_ have round-trip tests, not `B==map(A)` checks with pre-calculated values?

Reply:
testdata had to be updated to make the round trip tests pass.

Reply:
double checked, and yea this is fine:
- with this, tests fail before the fix because the value was not retained (yay better coverage)
- both with and without this, tests succeed after the fix, because the value is retained